src/Rules/Violations.php

```diff
@@ -117,6 +117,13 @@ static function (Violation $a, Violation $b): int {
         ));
     }
 
+    public function sort(): void
```

Comment:
could it be useful to add a simple test to document well this new behavior?
Do you think is useful @dbu ?

Reply:
yeah, agree that we should have a test for this. i added the test now.